src/components/Button.js

```diff
@@ -1,8 +1,8 @@
 import React from 'react';
-// eslint-disable-next-line import/no-extraneous-dependencies
 import PropTypes from 'prop-types';
 
-class Button extends React.Component {
+// eslint-disable-next-line import/prefer-default-export
```

Comment:
- [ ] Kindly remove the disable linters, as linter rules help guide us to write industry standard code.